src/utils/styled.js

```diff
@@ -3,8 +3,9 @@ const isTaggedTemplateLiteral = require('./tagged-template-literal').isTaggedTem
 /**
  * Check if something is a styled-components import declaration
  */
-const isStyledImport = node =>
-  node.type === 'ImportDeclaration' && node.source.value === 'styled-components'
+const isStyledImport = (node, moduleName) =>
+  node.type === 'ImportDeclaration' &&
+  node.source.value.split('/')[node.source.value.split('/').length - 1] === moduleName
```

Comment:
Yeah this is good :).

The only thing is maybe we should also do an `||` with `\` for Windows? Or maybe the `path` module can do something like this for us where it handles all the edge cases?